### DISCUSSION TITLE
Goal 10: Reduce inequality within and among countries

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#32] [General]
Goal 10: [Reduce inequality within and among countries](https://github.com/Project-Resilience/platform/blob/main/goals/goal_10.md)

**Comment by ofrancon** [comment]
Voice models for underserved languages are key to reduce inequalities in countries with a large illiterate population. This looks like a nice initiative: https://news.ycombinator.com/item?id=44950661

**Comment by GPaolo** [comment]
Paper from ICML 25:
- geographical bias in LLM: https://arxiv.org/pdf/2402.02680
- Bias evaluations in LLM: https://arxiv.org/pdf/2503.00093

**Comment by GPaolo** [comment]
Possibly related projects:
- Inclusive AI Lab: https://inclusiveailab.org/
- The inclusive AI: https://theinclusiveai.com/

**Comment by Allerbon** [comment]
Hi! Here’s the paper by Dana Kanze I mentioned: 'We Ask Men to Win and Women Not to Lose.' It’s a great deep dive into how systemic bias shapes the startup ecosystem. It specifically shows how investors ask men and women different types of questions, directly impacting the size of the funding they receive.

- https://gap.hks.harvard.edu/we-ask-men-win-and-women-not-lose-closing-gender-gap-startup-funding

- https://www.researchgate.net/publication/335679671_We_Ask_Men_to_Win_and_Women_Not_to_Lose_Closing_the_Gender_Gap_in_Startup_Funding

- https://www.hbs.edu/faculty/Pages/item.aspx?num=54068

And here’s some further evidence on the outperformance of women-led startups

- Boston Consulting Group https://www.bcg.com/publications/2018/why-women-owned-startups-are-better-bet
- TechCrunch https://techcrunch.com/2021/02/26/broaden-your-view-of-best-to-make-smarter-more-inclusive-investments/
- https://10years.firstround.com/#one
- IFC (World Bank)  1. Gender balanced leadership teams are correlated with approximately 25 percent
greater increases in valuation than unbalanced teams.  2. Imbalance in portfolio companies appears related to imbalance in General Partner investment teams. https://www.ifc.org/content/dam/ifc/doclink/2019/report-moving-toward-gender-balance-in-private-equity-and-venture-capital.pdf

Looking forward to continuing the conversation!

**Reply by ofrancon** [reply]
Great references, thanks! This discussion is very relevant for [SDG 5](https://github.com/Project-Resilience/platform/discussions/27) too, which is specifically about gender equality.

**Reply by ofrancon** [reply]
Sketched out project proposal [Reducing Investor Gender Bias in Startup Funding](https://github.com/Project-Resilience/platform/blob/main/proposals/investor_gender_bias.md) to capture this discussion. Data attributes are just place holders for now.

**Comment by ofrancon** [comment]
Regarding how LLMs reproduce inequalities and biases: 
[Unmanaged AI Adoption and the Risks to Enterprises](https://documents.trendmicro.com/assets/research-reports/unmanaged_ai_adoption.pdf): Evaluating regional biases, geofencing, data sovereignty, and censorship in LLM models
Authors: Josiah Hagen, Vladimir Kropotov, Robert McArdle, Fyodor Yarochkin

**Reply by ofrancon** [reply]
This can be discussed further in this thread: [Ensuring Equitable Global AI Access](https://github.com/Project-Resilience/platform/discussions/94)
